# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

BaseTrajectoryActionGoal action_goal
BaseTrajectoryActionResult action_result
BaseTrajectoryActionFeedback action_feedback
